src/App.js

```diff
@@ -1,25 +1,156 @@
-import logo from './logo.svg';
-import './App.css';
+
+import { useState } from "react";
+import { Routes, Route, useNavigate } from "react-router-dom";
+import axios from "axios";
+
+import "./App.scss";
+
+import Border from "./Components/Border";
+import Header from "./Components/Header";
+import UserSelectors from "./Components/UserSelectors";
+import DisplayParkInfo from "./Components/DisplayParkInfo";
+import DisplayToDos from "./Components/DisplayToDos";
+import Footer from "./Components/Footer";
+import ApiError from "./Components/ApiError";
+import Error404 from "./Components/Error404";
 
 function App() {
+  const [usersState, setUsersState] = useState("");
+  const [usersStateFull, setUsersStateFull] = useState("");
+  const [activities, setActivities] = useState([]);
+  const [inputError, setInputError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [parkInfo, setParkInfo] = useState([]);
+
+  const navigate = useNavigate();
+  const apiKey = "7XHElwOipPV6R4gzo3qbRAbY7q8MXA9TGPoKAHVX";
+
+  //stores usersState in stateful variable
+  const handleStateSelection = (e) => {
+    setUsersState(e.target.value);
+    setUsersStateFull(e.target.selectedOptions[0].label);
+  };
+
```

Comment:
I'm not 100% sure that I've understood this correctly so it may not apply _but_ i think your handleStateSelection and handleActivity selection could be put directly into your UserSelectors component if you wanted to shorten your App.js a little. I'm not sure if they are used elsewhere, but if they aren't you could throw em in UserSelectors and skip out on the passing the props!